torntools/scripts/js/globalFunctions.js

```diff
@@ -1491,6 +1491,9 @@ const STORAGE = {
 				hide_gym_highlight: false,
 				specialty_gym_1: "",
 				specialty_gym_2: "",
+				warn_when_stacking: true,
```

Comment:
Since this is a very invasive feature, I would set it tk false by default.

Reply:
Done.